.github/workflows/python-linters.yml

```diff
@@ -10,24 +12,53 @@ on:
 
 jobs:
   linters:
-    name: 🤖
+    name: >-
+      ${{ matrix.env.TOXENV }}
+      /
+      ${{ matrix.python-version }}
+      /
+      ${{ matrix.os }}
```

Comment:
nit:

```yml
    name: >-
      ${{ matrix.env.TOXENV }}-${{ matrix.python-version }} on ${{ matrix.os }}
```

Reply:
Not sure if that's shorter. My goal was to make it fit better in the sidebar while still communicating the description better.

We could even do something like this (spaceless):
```yaml
    name: >-
      ${{ matrix.env.TOXENV }}/${{ matrix.python-version }}@${{ matrix.os }}
```